### ISSUE TITLE
[每日信息流] 2025-02-23

### ISSUE THREAD

**Issue by M09Ic** [M09Ic/cryptor#9]
# 每日安全资讯（2025-02-23）

- BitMEX Blog
  - [ ] [Message to BitMEX Users: Another Hack, Another Wake Up Call](https://blog.bitmex.com/bybit-hack-message/)
- CoinDesk: Bitcoin, Ethereum, Crypto News and Price Data
  - [ ] [Ethereum 'Roll Back' Suggestion Has Sparked Criticism. Here's Why It Won't Happen](https://www.coindesk.com/tech/2025/02/22/ethereum-roll-back-suggestion-has-sparked-criticism-here-s-why-it-won-t-happen)
  - [ ] [Bybit Sees Over $4 Billion ‘Bank Run’ After Crypto’s Biggest Hack](https://www.coindesk.com/business/2025/02/22/bybit-sees-over-usd4-billion-bank-run-after-crypto-s-biggest-hack)
  - [ ] [Binance Research Survey Shows 95% of Latin American Crypto Users Plan to Buy More in 2025](https://www.coindesk.com/markets/2025/02/22/binance-research-survey-shows-95-of-latin-american-crypto-users-plan-to-buy-more-in-2025)
  - [ ] [Ether Price Spikes Further on Reports of Bybit Starting to Buy ETH](https://www.coindesk.com/markets/2025/02/22/ether-price-spikes-further-on-reports-of-bybit-starting-to-buy-eth)
  - [ ] [SEC Drops OpenSea Investigation Easing Pressure on NFT Market](https://www.coindesk.com/markets/2025/02/22/sec-drops-opensea-investigation-easing-pressure-on-nft-market)
  - [ ] [Ether Supply Squeeze? Bybit Hacker Emerges as World's 14th-Largest ETH Holder](https://www.coindesk.com/markets/2025/02/22/ether-supply-squeeze-bybit-hacker-emerges-as-world-s-14th-largest-eth-holder)
  - [ ] [Crypto Exchanges Start to Fill Bybit's $1.4B Hole as Hackers Move Stolen Funds](https://www.coindesk.com/markets/2025/02/22/exchanges-start-to-fill-bybit-s-usd1-4b-hole-as-hackers-move-stolen-funds)
  - [ ] [Arthur Hayes Floats the Idea of Rolling Back Ethereum Network to Negate $1.4B Bybit Hack, Drawing Community Ire](https://www.coindesk.com/markets/2025/02/22/arthur-hayes-proposes-rolling-back-ethereum-network-to-negate-usd1-4b-bybit-hack)
- Crypto Briefing
  - [ ] [Bybit fully restores withdrawal system following biggest crypto hack of all time—latest updates](https://cryptobriefing.com/bybit-withdrawal-restoration-after-hack/)
  - [ ] [Kanye West says he drops coin next week, claims all current ones are fake](https://cryptobriefing.com/kanye-west-crypto-token-launch/)
- Decrypt
  - [ ] [This Week in Bitcoin: Bybit Hack Rocks Market as ETFs Bleed—But Institutions Remain Bullish](https://decrypt.co/307119/this-week-bitcoin-bybit-hack-etfs-bleed-institutions-bullish)
  - [ ] [How to Prepare for Monad: The High-Speed EVM Layer-1 Blockchain](https://decrypt.co/resources/how-to-prepare-for-monad-the-high-speed-evm-layer-1-blockchain)
  - [ ] [Asteroid's Earth Impact Odds Plummet, Scientists Now Playing the Long Wait Game](https://decrypt.co/307310/asteroids-earth-impact-odds-plummet)
  - [ ] [Here Are Today's ‘Hamster Kombat: GameDev Heroes’ Daily Combo and Cipher Code](https://decrypt.co/resources/todays-hamster-kombat-daily-combo-cipher-code)
  - [ ] [Bybit Offers Bounty After $1.4 Billion in Ethereum Swiped in Biggest Crypto Hack Ever](https://decrypt.co/307359/bybit-bounty-ethereum-biggest-crypto-hack-ever)
  - [ ] [OpenSea Says SEC Will End Investigation Into Ethereum NFT Marketplace](https://decrypt.co/307352/opensea-sec-end-investigation-nft-marketplace)
  - [ ] [Why Bitcoin's Biggest Meme Coin Just Bridged Over to Solana](https://decrypt.co/307317/why-bitcoin-biggest-meme-coin-bridged-solana)
  - [ ] [Altcoins begin to Send, SEC to drop Coinbase Lawsuit, KAITO & IP soar!](https://decrypt.co/videos/interviews/B8mLqtaG/altcoins-begin-to-send-sec-to-drop-coinbase-lawsuit-kaito-ip-soar)
- ChainFeeds精选
  - [ ] [ChainFeeds Newsletter 每日精选 Web3 深度投研「简报」+ AI 驱动的热点新闻榜单，帮你做出聪明决策](https://substack.chainfeeds.xyz/p/vitalik-ef-glassnodeaptos-monad)
- 動區動趨-最具影響力的區塊鏈新聞媒體
  - [ ] [假的！Vitalik發起投票讓以太坊「回滾」拯救Bybit被駭，並非真實消息](https://www.blocktempo.com/rumor-that-vitalik-buterin-will-vote-for-ethereum-rollback-is-fake-news/)
  - [ ] [馬斯克恐引爆黃金「黑天鵝」，分析師：DOGE調查黃金儲備或推動資金流向比特幣](https://www.blocktempo.com/trump-plans-to-review-gold-reserves-at-fort-knox/)
  - [ ] [加密史上最大竊案》Bybit交易所14.7億美元為何遭竊、兇手是誰、有用戶擠兌風險？](https://www.blocktempo.com/the-largest-crypto-heist-in-history-why-did-bybit-lose-1-47-billion/)
  - [ ] [三箭Zhu Su：Bybit駭客事件的「ETH恐慌做空敘事」或將推動以太坊價格創新高](https://www.blocktempo.com/zhu-su-bybit-hacks-eth-panic-short-narrative-may-drive-ethereum-to-ath/)
  - [ ] [華爾街投行Bernstein：比特幣今年上看20萬鎂，加密貨幣牛市將持續到2026年](https://www.blocktempo.com/bernstein-expects-crypto-bull-run-to-continue-until-2026/)
  - [ ] [CZ決定進迷因市場「打土狗」：幣安限制員工炒幣反而與社群脫節](https://www.blocktempo.com/cz-says-he-plans-to-buy-memecoin/)
  - [ ] [Bybit內部出包？安全專家：北韓駭客Lazarus疑似入侵交易所員工電腦取得錢包多簽權限](https://www.blocktempo.com/lazarus-suspected-of-hacking-exchange-staff-to-access-multi-sign-wallets/)
  - [ ] [動區週報》Bybit 遭駭15億美元ETH、Pi 幣上線主網、FTX 正式還款…](https://www.blocktempo.com/quick-look-at-this-week-market-dynamics-and-analysis-0222/)
  - [ ] [Figure Markets推出首個「SEC批准」生息穩定幣YLDS，但有一限制條件](https://www.blocktempo.com/figure-markets-launches-interest-bearing-stablecoin-ylds/)
  - [ ] [Bybit駭客盜取14.7億鎂ETH，超越Vitalik、以太坊基金會成為「全球第14大」持有者](https://www.blocktempo.com/bybit-hacker-stole-1-47b-in-eth-now-the-14th-largest-holder-surpassing-vitalik-and-ethereum-foundation/)
- Cointelegraph.com News
  - [ ] [Here’s what happened in crypto today](https://cointelegraph.com/news/what-happened-in-crypto-today?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Bybit CEO discusses possibility of Ethereum blockchain rollback](https://cointelegraph.com/news/bybit-ceo-discusses-possibility-ethereum-blockchain-rollback?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Bybit expresses deep gratitude for industry support amid $1.4B hack](https://cointelegraph.com/news/bybit-expresses-deep-crypto-industry-support?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Security execs weigh in on ‘staggering’ scale of record Bybit hack](https://cointelegraph.com/news/bybit-hack-security-execs-weigh-in?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Lazarus Group moves funds to multiple wallets as Bybit offers bounty](https://cointelegraph.com/news/lazarus-moves-funds-multiple-wallets-bybit-offers-bounty?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [In pictures: Bybit’s record-breaking $1.4B hack](https://cointelegraph.com/news/bybit-1-4b-hack-in-pictures?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Blockchain’s next big breakthroughs: What to watch](https://cointelegraph.com/news/blockchain-s-next-big-breakthroughs?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Bybit hack, withdrawals top $5.3B, but ‘reserves exceed liabilities’ — Hacken](https://cointelegraph.com/news/bybit-hack-withdrawals-5-3b-reserves-secure-hacken?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Lazarus Group consolidates Bybit funds into Phemex hacker wallet](https://cointelegraph.com/news/lazarus-group-consolidates-bybit-phemex-hacker-wallet?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Is Bitcoin a good investment for retirement?](https://cointelegraph.com/explained/is-bitcoin-a-good-investment-for-retirement?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Why is Ethereum (ETH) price down today?](https://cointelegraph.com/news/why-is-ethereum-eth-price-down-today?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Bybit exploit exposes security flaws in centralized crypto exchanges](https://cointelegraph.com/news/bybit-hack-centralized-exchange-security-flaws?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Bybit processes all withdrawals, system returns to ‘normal pace’ — Ben Zhou](https://cointelegraph.com/news/crypto-exchange-bybit-withdrawals-processed-after-hack-ben-zhou?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [Blockchain gaming market is a ‘game of musical chairs’ — Gunzilla exec](https://cointelegraph.com/news/blockchain-gaming-capital-rotation-musical-chairs-gunzilla-games-interview?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
  - [ ] [SEC drops investigation into NFT marketplace OpenSea](https://cointelegraph.com/news/sec-closes-investigation-nft-platform-opensea?utm_source=rss_feed&utm_medium=rss&utm_campaign=rss_partner_inbound)
- HackerNoon
  - [ ] [The HackerNoon Newsletter: The Secret Hidden Power of Questions (2/22/2025)](https://hackernoon.com/2-22-2025-newsletter?source=rss)
  - [ ] [O.XYZ Launches OCEAN – Cerebras-Powered AI Engine, 10x Faster Than ChatGPT](https://hackernoon.com/oxyz-launches-ocean-cerebras-powered-ai-engine-10x-faster-than-chatgpt?source=rss)
  - [ ] [The Secret Hidden Power of Questions](https://hackernoon.com/the-secret-hidden-power-of-questions?source=rss)
  - [ ] [AI Models Are Learning to Prioritize Their Thoughts—And It’s Wildly Effective](https://hackernoon.com/ai-models-are-learning-to-prioritize-their-thoughtsand-its-wildly-effective?source=rss)
  - [ ] [What If AI Could Skip the Boring Parts? Google Researchers Just Made It Happen](https://hackernoon.com/what-if-ai-could-skip-the-boring-parts-google-researchers-just-made-it-happen?source=rss)
  - [ ] [This Clever AI Hack Could Cut Processing Costs in Half](https://hackernoon.com/this-clever-ai-hack-could-cut-processing-costs-in-half?source=rss)
  - [ ] [New AI Method Lets Models Decide What to Think About](https://hackernoon.com/new-ai-method-lets-models-decide-what-to-think-about?source=rss)
  - [ ] [Google Researchers Develop New AI Tech That Doesn't Waste Brainpower on Useless Words](https://hackernoon.com/google-researchers-develop-new-ai-tech-that-doesnt-waste-brainpower-on-useless-words?source=rss)
  - [ ] [The TechBeat: How to Fit an Elephant in a Spreadsheet (2/22/2025)](https://hackernoon.com/2-22-2025-techbeat?source=rss)
  - [ ] [A Brief Review of the Lie Group and the Geometries of SPD Manifolds](https://hackernoon.com/a-brief-review-of-the-lie-group-and-the-geometries-of-spd-manifolds?source=rss)
  - [ ] [A Lie Group Approach to Riemannian Batch Normalization](https://hackernoon.com/a-lie-group-approach-to-riemannian-batch-normalization?source=rss)
  - [ ] [Appendices on Our Dark Matter Study](https://hackernoon.com/appendices-on-our-dark-matter-study?source=rss)
  - [ ] [Dark Matter Study: Examining Several Models of WIMP](https://hackernoon.com/dark-matter-study-examining-several-models-of-wimp?source=rss)
  - [ ] [Dark Matter Analysis: A Lifetime Study](https://hackernoon.com/dark-matter-analysis-a-lifetime-study?source=rss)
  - [ ] [10 Mistakes Beginning AI Practitioners Make That You Should Avoid](https://hackernoon.com/10-mistakes-beginning-ai-practitioners-make-that-you-should-avoid?source=rss)
  - [ ] [The Most Effective Way to Run a Sprint Retrospective](https://hackernoon.com/the-most-effective-way-to-run-a-sprint-retrospective?source=rss)
- Release notes from sui
  - [ ] [sui_v1.44.0_1740275216_ci](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740275216_ci)
  - [ ] [sui_v1.44.0_1740270040_rel_notes](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740270040_rel_notes)
  - [ ] [sui_v1.44.0_1740263616_ci](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740263616_ci)
  - [ ] [sui_v1.44.0_1740258296_rel_notes](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740258296_rel_notes)
  - [ ] [sui_v1.44.0_1740220550_ci: [chore] Remove RUSTSEC-2024-0421 (#21317)](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740220550_ci)
  - [ ] [sui_v1.44.0_1740215076_rel_notes: [chore] Remove RUSTSEC-2024-0421 (#21317)](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740215076_rel_notes)
  - [ ] [sui_v1.44.0_1740210066_ci: [faucet] Try to find an address with sufficient gas coins (#21302)](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740210066_ci)
  - [ ] [sui_v1.44.0_1740204344_rel_notes: [faucet] Try to find an address with sufficient gas coins (#21302)](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740204344_rel_notes)
  - [ ] [sui_v1.44.0_1740198706_ci](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740198706_ci)
  - [ ] [sui_v1.44.0_1740193598_rel_notes](https://github.com/MystenLabs/sui/releases/tag/sui_v1.44.0_1740193598_rel_notes)
